ForumProject/notifications/models.py

```diff
@@ -54,4 +54,15 @@ class Meta:
         # Add model-level constraint
         constraints = [
             models.CheckConstraint(check=models.Q(project__isnull=False) | models.Q(investor__isnull=False), name='project_or_investor_required'),
-        ]
\ No newline at end of file
+        ]
+
+class NotificationPreferences(models.Model):
+    startup = models.OneToOneField(Startup, 
+        on_delete=models.CASCADE,
+        related_name='notice_preferences',
+        null=True
+    )
+    email_on_followers_change = models.BooleanField(default=True, verbose_name='Followers change email notices')
+    email_on_share_subscription = models.BooleanField(default=True, verbose_name='Sibscriptions change email notices')
```

Comment:
There's a minor spelling error in "Sibscriptions" which should be "Subscriptions"

Reply:
Thank you, corrected